### ISSUE TITLE
Delete card button not showing up - CSS issue?

### ISSUE THREAD

**Issue by rodobodolfo** [rcdexta/react-trello#177]
Hello! Thanks for your work!

**Describe the bug**
Delete button not showing up: 

the css for the button  has opacity 0,
```
.ejXrMv {
    position: relative;
    display: inline-block;
    width: 4px;
    height: 4px;
    opacity: 0;
    overflow: hidden;
    border: 1px solid #83bd42;
    border-radius: 50%;
    padding: 4px;
    background-color: #83bd42;
}
```

Tried `hideCardDeleteIcon={false}` with no success

**To Reproduce**
1. Go to https://rcdexta.com/react-trello/?selectedKind=Editable%20Board&selectedStory=Add%2FDelete%20Cards&full=0&addons=0&stories=1&panelRight=0

**Comment by rcdexta** [comment]
Fix on the way. Will be available in next release in few days.

**Comment by rcdexta** [comment]
Should be fixed in 2.0.10
